packages/libs/api/icy-graphql-client/src/utils/removeNodesAndEdges.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @todo implement generic typing
+ */
+export const removeNodesAndEdges = (data: any): any | any[] => {
```

Comment:
Nice!!

Reply:
have fun typing this hehe

Reply:
yeah haha, we'll have to get back to it

Reply:
it's probably not really that necessary since i don't think you can ever use the output of this method in a context that retains types